docs/guides/documentation-style.stories.mdx

```diff
@@ -0,0 +1,87 @@
+import { Meta } from "@storybook/addon-docs";
+import showDo from "./assets/Atlantis_Docs_Show_Do.png";
```

Comment:
This feels like a weird way to import images but it's how it's done in `/guides/create-basic-component.stories.mdx` and it seems to work.

Reply:
It's how image imports work with modern bundlers. You just import the image from wherever and they handle making sure it's available in the browser!